module/zfs/zil.c

```diff
@@ -3140,18 +3105,14 @@ zil_commit_waiter_timeout(zilog_t *zilog, zil_commit_waiter_t *zcw)
 		 *   because it's blocked trying to acquire the waiter's
 		 *   lock, which occurs prior to calling dmu_tx_commit()
```

Comment:
This comment seems a little odd here, now that the `mutex_exit()` has been removed. Perhaps move (and adapt it) above to explain why the `zcw_lock` is being dropped there.

Reply:
Thanks. I've removed this comment and written new one above.